macosx/Controller.mm

```diff
@@ -5042,12 +5021,27 @@ - (void)updateWindowAfterToolbarChange
             [self removeStackViewHeightConstraints];
         }
 
+        //this fixes a macOS bug where on toggling the toolbar item bezels will show
+        [self hideToolBarBezels:YES];
+
         dispatch_async(dispatch_get_main_queue(), ^{
             [self setWindowSizeToFit];
+            [self hideToolBarBezels:NO];
         });
     }
 }
 
+- (void)hideToolBarBezels:(BOOL)hide
```

Comment:
But why hide them?

Reply:
other =wise this occurs #1903

Reply:
But without bezels toolbar looks unnatural :(

Reply:
they are restored immediately

Reply:
Ah, okay then

Reply:
see here:
```
[self hideToolBarBezels:YES];

        dispatch_async(dispatch_get_main_queue(), ^{
            [self setWindowSizeToFit];
            [self hideToolBarBezels:NO];
        });
```

Reply:
more workarounds to fix Cocoa/AppKit bugs...